chart/templates/mayastor/obs/stats-service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-stats-aggregator
+  labels:
+    app: stats-aggreagtor
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+spec:
+  ports:
+    - name: stats
+      port: 9505
```

Comment:
I think the port should be 9090. It was decided that 9090 is to be used to expose the stats.